lib/facility.rb

```diff
@@ -1,14 +1,62 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, 
+              :address, 
+              :phone, 
+              :services, 
+              :registered_vehicles,
+              :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
-    @services = []
+  def initialize(name:, address:, phone:)
```

Comment:
I believe we would want to pass in a hash because it is a hash?

Reply:
Yes, since it is coming in as a hash, you would pass a single argument to you initialize
```ruby
def initialize(facility_info)
  @name     = facility_info(:name]
  @address  = facility_info[:address]
  @phone    = facility_info[:phone]
...
end
```